test/metamodelutil.js

```diff
@@ -186,6 +186,16 @@ describe('importFullyQualifiedNames', () => {
             const result = MetaModelUtil.importFullyQualifiedNames(ast);
             result.should.deep.equal(['test.Foo', 'test.Bar']);
         });
+        it('should return imports when aliasing', async () => {
+            const ast = {
+                $class: 'concerto.metamodel@1.0.0.ImportTypes',
+                namespace: 'test',
+                types: ['Foo', 'Bar'],
+                aliasedTypes:{'f':'Foo','b':'Bar'}
+            };
+            const result = MetaModelUtil.importFullyQualifiedNames(ast);
+            result.should.deep.equal(['test.Foo', 'test.Bar']);
```

Comment:
I'm not sure if this test does anything with regards to import aliasing. I guess it ensures that the change is not a breaking change, but it's not making use of `aliasedTypes`.

It is testing `MetamodelUtil`. So something like [this test](https://github.com/accordproject/concerto-metamodel/blob/7aba6de472a712d83e7c135b255bf4e084b7f541/test/metamodelutil.js#L53) would be a better option, since it's testing if local names are resolvable. You might need to update the `MetamodelUtil` too, which is good, then the you'd be taking advantage of test driven development.

Reply:
Yes, I felt the same. 

Will have to add a function for resolving local types in the `MetamodelUtil` and then some tests based on it. Both good and bad ones can be added.

Reply:
@ekarademir 

I have updated the MetamodelUtil and handled the function resolveTypes for the aliased import case.